apps/super-app/src/routes/index.tsx

```diff
@@ -1,3 +1,3 @@
 export { LandingPage } from './LandingPage';
 export { Options } from './Options';
-export { SupportDetailWip } from './SupportDetailWip';
+export { SupportDetailPage } from './SupportDetailPage';
```

Comment:
Since we aren't using a generic `SupportDetailPage` any more, perhaps we could omit this change for now and reintroduce it when it makes sense.

Reply:
>  If we are going to have individual pages, we can rename them to specific providers later?

Yes. I meant that we would rename them to e.g. `SupportTwc2`.

> I just feel that the "WIP" term is misleading. It's not really "WIP" when you finish one page, no?

Yes, we'll get rid of it when we are done by renaming to `SupportTwc2`. If we are done now, we can make that change.